tests/rptest/utils/si_utils.py

```diff
@@ -58,6 +58,58 @@ def to_ntp(self) -> NTP:
         return NTP(self.ns, self.topic, self.partition)
 
 
+@dataclass
+class LogRegionSize:
+    total: int
+    accessible: int
```

Comment:
these can be set to 0 by default, so that `__init__` does not need to be defined, eg

```
    total: int = 0
    accessible: int = 0
```